rush/rush-lib/src/utilities/Utilities.ts

```diff
@@ -142,6 +143,16 @@ export default class Utilities {
   }
 
   /**
+   * return all files in the folder recursively through all sub-directories.
```

Comment:
> return all files in the folder recursively through all sub-directories. [](start = 4, length = 72)

How useful is this "reusable" utility?  Would it be simpler to use the glob.sync() API instead?